### PR TITLE
Pluggable Hedera Consensus Service for Hyperledger Fabric

### PR DESCRIPTION
**Pull request by donaldthibeau**
## Checklist for Hyperledger Lab Proposal PR

The Pluggable HCS Lab proves the ability of a Hyperledger Fabric network to connect to the Hedera Consensus Service running on a publicly accessible testnetwork.

### CODE REVIEW

**Comment by tkuhrt** [comment]
@donaldthibeau : Can you follow the steps [here](https://github.com/hyperledger-labs/hyperledger-labs.github.io/pull/119/checks?check_run_id=519460301) to fix the DCI issues.

**Comment by tkuhrt** [comment]
@christo4ferris : Can you confirm sponsorship?

File: labs/pluggable-hcs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Lab Name
+_Pluggable Hedera Consensus Service_
+
+# Short Description
+_The Pluggable HCS Lab proves the ability of a Hyperledger Fabric network to connect to the Hedera Consensus Serivce running on a publicly accessilbe testnetwork._
```
Comment (tkuhrt) [inline labs/pluggable-hcs]: Typo `accessilbe` should be `accessible`

File: labs/pluggable-hcs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Lab Name
```
Comment (tkuhrt) [inline labs/pluggable-hcs]: For the entire file, you can remove the `_` underscores from the text. The _italics_ in the original template were the instructions so you would know what to do.

File: labs/pluggable-hcs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+_Pluggable Hedera Consensus Service_
+
+# Short Description
+_The Pluggable HCS Lab proves the ability of a Hyperledger Fabric network to connect to the Hedera Consensus Serivce running on a publicly accessilbe testnetwork._
+
+# Scope of Lab
+_This lab is designed to enable a pluggable consensus option within Hyperledger Fabric. The Hedera Consensus Service is available on the Hedera public network. The Lab initially uses the Build Your First Network (BYFN) Hyperledger Fabric sample to connect a set of Orderers to the Hedera Consensus Service. The demo proves the ability of HCS to deliver a set of ordered transactions from the public network to a Fabric network for block creation and propogation._
+
+# Initial Committers
+_Enter the Github IDs for the set of initial committers._
```
Comment (tkuhrt) [inline labs/pluggable-hcs]: You can remove this instruction line.

File: labs/pluggable-hcs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+- https://github.com/xin-hedera
+- https://github.com/mikeburragejr
+- https://github.com/donaldthibeau
+
+# Sponsor
+_Provide the name of your sponsor. A sponsor is required and must be a maintainer of one of the Hyperledger projects, a TSC member, or a WG chair._
```
Comment (tkuhrt) [inline labs/pluggable-hcs]: You can remove this instruction line.

File: labs/pluggable-hcs [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+- https://github.com/christo4ferris - Hyperledger Technical Steering Committee Member
+
+# Pre-existing repository
+_If you currently have a Github repository that you wish to transfer to the Hyperledger Labs organization, please provide a link here. **NOTE: Please refer to the README for additional information on existing repositories.**_
```
Comment (tkuhrt) [inline labs/pluggable-hcs]: You can remove this instruction line.

File: labs/pluggable-hcs [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+- https://github.com/hashgraph/fabric-samples-hcs/tree/master/first-network
```
Comment (tkuhrt) [inline labs/pluggable-hcs]: Are you saying you want to move the entire https://github.com/hashgraph/fabric-samples-hcs repository to Hyperledger Labs? Secondly, can you confirm that this repository has DCO signoff on all commits?

Comment (xin-hedera) [reply labs/pluggable-hcs]: Yes, we need the entire repo. In addition, we need the following repo as well:
https://github.com/hashgraph/fabric-hcs

The commits don't have DCO signoff. I'll start working on that.

File: labs/pluggable-hcs (hunk 1 above)
Comment (lehors) [reply labs/pluggable-hcs]: Serivce -> Service

Comment (lehors) [inline labs/pluggable-hcs]: I suggest using "demonstrates" rather than "proves".

Comment (xin-hedera) [reply labs/pluggable-hcs]: just made the change

File: labs/pluggable-hcs (hunk 6 above)
Comment (xin-hedera) [reply labs/pluggable-hcs]: all the commits in fabric-samples-hcs and fabric-hcs have my DCO signoff now.

**Comment by xin-hedera** [comment]
> Can these two repos be combined into one? or is the expectation that there will be two separate repos in labs:
> 
> * pluggable-hcs
> * pluggable-hcs-samples
> 
> My preference would be a single repo with a subdirectory for samples.

I'll merge first-sample into fabric-hcs repo so we just need the pluggable-hcs repo in labs.

**Comment by trbs** [comment]
Think we are waiting on @christo4ferris to confirm sponsorship

**Comment by dcmiddle** [comment]
@vipinsun  did you get @christo4ferris confirmation? I don't see it in the comment thread of this PR and the PR reviewers panel doesn't indicate his approval.

**Pull request merged**